spacy_llm/tests/backends/test_dolly.py

```diff
@@ -34,7 +34,7 @@
 """
 
 
-@pytest.mark.parametrize("backend", ("spacy.Dolly_HF.v1", "spacy.DollyHF.v1"))
+@pytest.mark.parametrize("backend", ("spacy.Dolly_HF.v1", "spacy.Dolly_HF.v1"))
```

Comment:
Why this edit?

Reply:
Unintentional. Fixed in https://github.com/explosion/spacy-llm/pull/151/commits/2732d5993f7afd002270d036731d07d9c0504a22.